src/views/HomeView.vue

```diff
@@ -1,71 +1,76 @@
 <template>
   <div>
-    <b-container class="kiaora">
-      <h1>Kia ora {{ username }}</h1>
-      <dl>
-        <dt>Username</dt>
-        <dd id="username">{{ username }}</dd>
-        <dt v-show="firstname">First name</dt>
-        <dd
-          v-show="firstname"
-          id="firstname">{{ firstname }}
-        </dd>
-        <dt v-show="lastname">Last name
-        </dt>
-        <dd
-          v-show="lastname"
-          id="lastname">{{ lastname }}
-        </dd>
-        <dt>Email</dt>
-        <dd id="email">{{ email }}</dd>
-        <dt>Your groups</dt>
-        <dd id="groups">
-          <span
-            v-for="(group, index) in groups"
-            :key="index">
-            <b-link :to="{ name: 'group', params: { groupname: group.groupname }}">
-              {{ group.groupname }}</b-link><span v-if="index+1 !== groups.length">, </span>
-          </span>
-          <spinner :fetching="!fetched"/>
-          <span v-if="fetched && !groups.length">
-            <p>
-              <icon-link
-                :icon="['fas', 'exclamation-triangle']"
-                :link="{ name: 'devices'}"/>
-              You will not see any results as you do not have any devices (cacophonators or cacophonometers)</p>
-            <p>If you are setting up your own device you should  <b-link :to="{ name: 'groups'}">create a group </b-link>for your devices.</p>
-            <p>Otherwise ask the admin of your device for access to it.</p>
-          </span>
-        </dd>
-      </dl>
-    </b-container>
     <Hero/>
+    <b-container class="card-container">
```

Comment:
Might have been clearer to have a single v-if="hasGroups" here because the screen content is quite different.   And maybe a component for the  you are new here.   

You don't need to fix it now though.   Just a thought.